Adjust login and sign up

Oozaru71 opened the pull request:
Made quite a few changes but i left a summary of the most noticeable ones but now everything should be as expected,

Commit:
MajorChanges summarized:
*Names of ActionResults were updated to match new view folder
*New folder added to home folder to organize sign in pages
*delted some comments from some js files that were causing a small error
*created a new css folder that contains all cs and pictures for all other pages to use
*start up was modified to fit new view names
*signing and sign up work as intended in the appropriate view
*homepages shows user name and picture

Alejandro-HUB commented on Views/Home/Index.cshtml:
```diff
@@ -1,91 +1,1358 @@
+        </a>
+        <a href="index.html" class="navbar-brand text-lt">
+          <i class="icon-earphones"></i>
+          <img src="/css2/images/logo.png" alt="." class="hide">
```
I mentioned them on the video but I’ll write it here too.

1) The logo on the top left
2) Logout button
3) Positioning of the music bar
4) Positioning of the picture on the left panel

Commit:
Most of the issues youu mentioned should be fixed
There is one issue that could not be completely resolved and that is the logo. The reason it may look cut off is because the image we are imputting for our profile pic is resizing the container is is on and cutting off a part of the UI. I need to experiment further to see how to fix it.

Alejandro-HUB commented on Views/Home/Index.cshtml:
```diff
@@ -118,7 +121,7 @@
               <li class="divider"></li>
               <li>
-                 <button type="button" class="logoutbtn" href="#" onclick="signOut()" id="logout">LOG OUT</button>
+                <a href="modal.lockme.html" data-toggle="ajaxModal" >Logout</a>
```
To add the logic for the logout button you need to go inside the ajax modal that gets triggered once you click your name. You will find the logout button inside modal.lockme.html.

Also, you do not need to create another pull request when you push your changes the pull request gets automatically updated. Which is why I am denying your other pull request.

Finally, the logo is a css issue all you need to do is look at which is one has css that messes up the other. If you think its the user picture then look into the user picture's css and make it smaller or play around with it.

Commit:
Logout functionality works like previous muzilla version, it does not icorporate the modal.locke.html simply logs using actionresult, size for avatar fixed

Oozaru71 commented:
I have fixed both problems, log out and avatar pic size.

Alejandro-HUB reviewed (approved):
Great job, it's looking good. I will assign a new task to you tomorrow 👍
